codeforces/540a_combination_locks.cpp

```diff
@@ -1,11 +1,13 @@
+Problem Statement: McDuck keeps his most treasured savings in a home safe with a combination lock. Each time he wants to put there the treasures that he's earned fair and square, he has to open the lock.The combination lock is represented by n rotating disks with digits from 0 to 9 written on them. Scrooge McDuck has to turn some disks so that the combination of digits on the disks forms a secret combination. In one move, he can rotate one disk one digit forwards or backwards. In particular, in one move he can go from digit 0 to digit 9 and vice versa. What minimum number of actions does he need for that?
+	

```

Comment:
Hi Aman, thanks for the PR.
However, you have made the CPP file un-compilable. Please fix this.

Reply:
I have made changes in the cpp file and now it is compiling properly.
Thanks for pointing it out.